common/src/main/java/owmii/powah/block/reactor/Builder.java

```diff
@@ -51,6 +51,13 @@ else if (!this.queue.isEmpty()) {
                 while (itr.hasNext()) {
                     BlockPos pos = itr.next();
                     BlockState state = this.reactor.getBlock().defaultBlockState();
+                    if (!world.getBlockState(pos).getMaterial().isReplaceable()) {
+                        final List<BlockPos> placed = new ArrayList<>(this.getPosList());
+                        placed.removeAll(this.queue);
+                        placed.forEach(b -> world.destroyBlock(b, true));
+                        world.destroyBlock(this.reactor.getBlockPos(), true);
```

Comment:
Tried that and it caused dupes, probably possible to fix demolish so that doesn't happen, but the call stack gets kind of deep and I didn't feel it was justified to spend time on figuring that out when the current solution works.

Reply:
Does it work if you add `list.removeAll(this.queue);` in `demolish`? The two ways to break blocks (`setBlockState` vs `destroyBlock`) could create some extra maintenance.